src/main/java/com/google/sps/models/PCAPAnalysis.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.models;
+
+public class PCAPAnalysis {
```

Comment:
if you can make this an interface it will be more extensible

Reply:
Fixed.